tutorials/navigation/real_time_navigation_3d.rst

```diff
@@ -39,7 +39,7 @@ navigation mesh above the areas in your scene that can be navigated 
 .. image:: img/baked_navmesh.png
 
 Keep in mind that the navmesh shows where the center of an entity can
-go. For example, if you set the agent radius to 1 then your
+go. For example, if you set the agent radius to .5 then your
 navigation mesh will have a distance of .5 from any ledges or walls
```

Comment:
While we're at it, I'd prefer using explicit leading zero to look less confusing at a distance:

```suggestion
go. For example, if you set the agent radius to 0.5 then your
navigation mesh will have a distance of 0.5 from any ledges or walls
```